README.md

```diff
@@ -86,6 +86,7 @@ The workflow in the use case shown below includes data cleaning, ML model traini
 
 
 ## Publications (Interdisciplinary):
+* (2/2025) DS4ALL: Teaching High-School Students Data Science and AI/ML Using the Texera Workflow Platform as a Service, Jiadong Bai, Xiaozhen Liu, Anthony Cuturrufo, Alexander Kundu Taylor, Jeehyun Hwang, Mingyu Derek Ma, Xinyuan Lin, Yanqiao Zhu, Yicong Huang, Yunyan Ding, Wei Wang and Chen Li, to appear in Data Science Education K-12: Research to Practice Annual Conference 2025, San Antonio, Texas [Conference website](https://web.cvent.com/event/d641bd9f-6c99-4cbc-951b-33b1ca05d4ed/summary).
```

Comment:
just put the link on the conference name.

Reply:
do you have a camera ready version of the paper? if so please put the pdf. otherwise don't put it.

Reply:
We don't have one